src/codegen/math/float/tanh.rs

```diff
@@ -0,0 +1,112 @@
+//! Vertical floating-point `tanh`
+#![allow(unused)]
+
+// FIXME 64-bit 1 elem vectors tanh
+
+use libm::{
+    F32Ext,
+    F64Ext
+};
```

Comment:
I think it would be better to move these traits down to the functions that implement them.